project/FrontEnd/collaborative_science_platform/lib/models/workspaces_page/workspace.dart

```diff
@@ -39,20 +44,26 @@ class Workspace {
     List<Node> references = referencesList.map((e) => Node.fromJsonforNodeDetailPage(e)).toList();
 
     String statusString = jsonString['status'];
-    WorkspaceStatus status = (statusString == "finalized") ? WorkspaceStatus.finalized
-        : (statusString == "workable") ? WorkspaceStatus.workable
-        : (statusString == "in_review") ? WorkspaceStatus.inReview
-        : (statusString == "published") ? WorkspaceStatus.published
-        : WorkspaceStatus.rejected;
-
+    WorkspaceStatus status = (statusString == "finalized")
```

Comment:
Just a reminder for the future projects, nested ternary operators can be hard to read and understand, especially when they are long and complex. They can also lead to errors and bugs if not written carefully.

So I would prefer switch case here